src/os/bluestore/NVMEDevice.cc

```diff
@@ -801,6 +816,7 @@ void io_complete(void *t, const struct spdk_nvme_cpl *completion)
   assert(queue != NULL);
   assert(ctx != NULL);
   ++queue->completed_op_seq;
+  --queue->current_queue_depth;
```

Comment:
I understand now... because of the spike lib limit, submit and poll should be in the same thread...